js-core/homeworks/homework-3/src/main.js

```diff
@@ -0,0 +1,153 @@
+    /*
+    Напишите функцию которая будет возвращать объект
+    с свойством name, а значением будет первый аргумент функции
+    */
+
+    function returnFunction(param){
+        let obj = {
+            name: param
+        }
+        return  obj
+    }
+    let someObj = returnFunction('Aloe vera');
+    console.log(someObj)
+
+
+//===================================================================
+
+    /*
+    Функция будет принимать 1 аргумент - ОБЪЕКТ у которого
+    будет свойство name
+    и возвращать новый объект изменяя свойство name
+    в большой регистр
+    */
+    let myObj = {
+        name: 'Some object name'
+    }
+
+    function returnObj (obj){
+        let modifyObj = {
+            name: String(obj.name).toUpperCase()
```

Comment:
Not sure we have to use *String* here, it's a bit overhead. 

I will talk about it in classrom